app/views/groups/show.html.erb

```diff
@@ -0,0 +1,67 @@
+<nav class="header">
+  <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+    <li>
+    <a href="/"><i class="fa fa-arrow-left"></i></a>
+    </li>
+    <li>
+      <h2 class="heading fw-bold">Category Details</h2>
+    </li>
+    <li>
+    <%= link_to destroy_user_session_path, method: :delete do %>   
+      <i class="fa fa-sign-out"></i>
+    <%end%>
+    
+   
+    </li>
+  </ul>
+  </nav>
+<%=render 'layouts/errornotice'%>
+<div class="p-2 category-div">
+  <div class="category-info">
+    <p> <%=image_tag @group.icon ,class:"p-2 m-2 rounded img-thumbnail"%></p>
+    <ul class="p-2 m-2">
+        <li> <p class="p-2 m-2 text-muted h2" ><%= @group.name%></p></li>
+        <li> <p class="p-2 m-2"> <%= @group.created_at%></p></li>
+    </ul>
+  </div>
+  <ul class="p-2 m-2">
+    <li><p class="p-2 m-2 h3"> $<%= @group.money_tracks.sum(:amount)%></p></li>
+    <li>
+    <%if can? :destroy, @group%>
+        <%=link_to "Delete", group_path(@group.id), method: :delete, class:"btn btn-primary px-4 m-2"%>
+    <%end%>
+    </li>
+  </ul>
+</div>
+
+
+ <% if @group.money_tracks.any? %>
+    
+
+    
+    <% @group.money_tracks.order(created_at:'DESC').each do |fd|%>
+      <div class="p-2 m-4 category-div">
+        <div class="category-info">
+          <ul class="p-2 m-2">
+            <li><p class="p-2 m-2 text-muted h2" ><%= fd.name%></p></li>
+            <li> <p class="p-2 m-2"> <%= fd.created_at%></p></li>
```

Comment:
- [ ] Please keep the `category details` page design similar to what you have given. Currently, you are using the icon for the category as well where the total amount is written, please remove that, Also keep the date format same as the design so that anyone can understand.

### Your design 

![Screenshot from 2022-01-05 20-13-43](https://user-images.githubusercontent.com/22434660/148238897-1c6fb2f2-d849-425b-9694-96ec1bac83cc.png)

### Given design

![Screenshot from 2022-01-05 20-13-25](https://user-images.githubusercontent.com/22434660/148238997-db6601e5-a939-440c-be3d-a8543224ca78.png)

Reply:
Hi, @Dipeshtwis  thank you for the review. For transactions, there is no image. So how can I show images for transactions

Reply:
Hi @ridaarif98, you understand me wrong, you don't have to add image but you have to remove the image, and just you have to mention the `category name` and `amount` at the top as given in the design where it is written `Gift credit`. Compare that and update accordingly.